File: tournaments/numberOfSolutions/numberOfSolutions.js
function numberOfSolutions(n) {

  var result = 0;
  for (var a = n + 1; a < 2 * n; a++) {
    if ((a * n) % (a - n) === 0) {
      result++;
    }
  }

  return result * 2 + 1;
}

File: tournaments/numberOfSolutions/numberOfSolutions.cpp
int numberOfSolutions(int n) {

  int result = 0;
  for (int a = n + 1; a < 2 * n; a++) {
    if ((a * n) % (a - n) == 0) {
      result++;
    }
  }

  return result * 2 + 1;
}
